Lib/defcon/objects/font.py

```diff
@@ -884,6 +893,8 @@ def _get_identifiers(self):
     # object as dirty.
 
     def _get_guidelines(self):
+        if self._info is None:
+            self.info
```

Comment:
Pity that we have to throw self.info away and use the getter for its obscure side effects.
Would `return list(self.info.guidelines)` achieve the same effect?